controllers/configmap.go

```diff
@@ -106,6 +107,14 @@ func (r *SingleClusterReconciler) createConfigMapData(rack *asdbv1.Rack) (
 
 	confData[aerospikeTemplateConfFileName] = confTemp
 
+	// This field value is rectified in 3.3.0.
```

Comment:
rack-id was historically set to 0 for all namespaces, but since the AKO 3.3.0, it reflects actual values. This change led to hash mismatches during the AKO 3.3.0 upgrade, triggering unnecessary warm restarts. Solution: Replace real rack-id with 0 in hash calculations to avoid this issue.